### ISSUE TITLE
Item details - text small, and does not adjust with Dynamic Type settings

### ISSUE THREAD

**Issue by mitchcohen** [kenstir/hemlock-ios#35]
Text on the Item Details screen is small.  This is subjective.  I suspect it is smaller than the HIG recommended Body size of 17, but haven't confirmed.  See https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/.

Further, the size here, and in most screens, does not adapt to Dynamic Type.

![IMG_6227](https://user-images.githubusercontent.com/102565/103462523-bdcc9f80-4cf3-11eb-94d5-d6cbe9e49c06.PNG)

**Comment by kenstir** [comment]
Good idea.

Note to self: https://developer.apple.com/documentation/uikit/uifont/scaling_fonts_automatically.

@mitchcohen are you interested in contributing pull requests to fix any of these issues?

**Comment by mitchcohen** [comment]
@kenstir Yes, I’m very interested in contributing to the project.

(I sent you an email earlier today detailing a build problem I was having.  If you don’t see that, or prefer I post the problem as an issue here, just let me know.)

**Comment by mitchcohen** [comment]
Notes on my pending commit for this issue.

All labels have been updated to support Dynamic Type.  Some labels have new (larger) default sizes:
Synopsis is now larger (15->17) and black instead of gray.
Button type larger (15->20), partially to get closer to 44pt button height.
Others the same or 1pt larger.
Title’s size is unchanged, but is now max 5 lines (up from 2).

Ideally I would have implemented automatic updates if the user adjusts their preferred size without refreshing the view.  I was unable to find a reliable way to do that.  I added a UIContentSizeCategory.didChangeNotification notification in XDetailsPagerViewController.swift, which caught the change, but I couldn’t find a decent way to refresh.  The closest I got was pagerNode.reloadData() which worked, but frequently locked the UI and eventually crashed.  I’m sure there’s a way to do this, but lack of support for this is something few (if any) will notice.
![Simulator Screen Shot - iPhone 12 mini - 2021-01-18 at 16 33 40](https://user-images.githubusercontent.com/102565/104977518-bda7f180-59cd-11eb-9c9f-97e64693ad85.png)
![Simulator Screen Shot - iPhone 12 mini - 2021-01-18 at 19 50 44](https://user-images.githubusercontent.com/102565/104977516-bc76c480-59cd-11eb-98da-5c0560758c06.png)
![Simulator Screen Shot - iPhone 12 mini - 2021-01-18 at 19 50 54](https://user-images.githubusercontent.com/102565/104977517-bd0f5b00-59cd-11eb-8fe5-a0151a88e706.png)
